yarn-project/circuits.js/src/abis/abis.ts

```diff
@@ -323,6 +323,7 @@ export function computeBlockHash(
  * Computes the globals hash given the globals.
  * @param globals - The global variables to put into the block hash.
  * @returns The globals hash.
+ * TODO: move this to GlobalVariables?
```

Comment:
Should this have an issue? 
While I think it would be fine to move it, the current way aligns more with what we are doing for other functions such as the block hash.

Reply:
> the current way aligns more with what we are doing for other functions such as the block hash.

I didn't realize that. I see that in plenty of the other cases in that file the functions have a single object on the input (e.g. `FunctionLeafPreimage`) so if we decide to do this we should probably refactor it all.

I think it's better to have the hash functions included with the classes as that is where they are the easiest to find.

Reply:
I agree with having the individual functions do it since it is easier to find, but is a more low prio task for the new year.